src/css/mapbox-gl.css

```diff
@@ -111,6 +110,30 @@
     background-color: rgba(0, 0, 0, 0.05);
 }
 
+.mapboxgl-ctrl-group > button:focus {
+    box-shadow: 0 0 2px 2px rgba(0, 150, 255, 1);
+}
+
+.mapboxgl-ctrl-group > button:focus-visible {
+    box-shadow: 0 0 2px 2px rgba(0, 150, 255, 1);
+}
```

Comment:
Nit: you can group these two selectors.

Reply:
I actually can't because I just realized Firefox uses the `-moz-focusring` selector instead of `focus-visible` so combining them makes the entire rule invalid in Firefox which breaks all focusing on the buttons. It turns out that Firefox and Safari don't appear to register clicks as focusing the button so they just work without this rule. I'll still leave it though because it's a progressive enhancement in Chrome and presumably newer versions of Edge.